docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -0,0 +1,605 @@
+| `serverStartPolicy` | Determines which WebLogic Server instances will be started. Legal values are `NEVER`, `IF_NEEDED`, `ADMIN_ONLY`. | `IF_NEEDED` |
+| `t3ChannelPort` | Port for the T3 channel of the NetworkAccessPoint. | `30012` |
+| `t3PublicAddress` | Public address for the T3 channel.  This should be set to the public address of the Kubernetes cluster.  This would typically be a load balancer address. <p/>For development environments only: In a single server (all-in-one) Kubernetes deployment, this may be set to the address of the master, or at the very least, it must be set to the address of one of the worker nodes. | If not provided, the script will attempt to set it to the IP address of the Kubernetes cluster |
+| `weblogicCredentialsSecretName` | Name of the Kubernetes secret for the Administration Server's user name and password. If not specified, then the value is derived from the `domainUID` as `<domainUID>-weblogic-credentials`. | `soainfra-domain-credentials` |
+| `weblogicImagePullSecretName` | Name of the Kubernetes secret for the Docker Store, used to pull the WebLogic Server image. |   |
+| `rcuSchemaPrefix` | The schema prefix to use in the database, for example `SOA1`.  You may wish to make this the same as the domainUID in order to simplify matching domains to their RCU schemas. | `SOA1` |
+| `rcuDatabaseURL` | The database URL. | `soadb.soans:1521/soapdb.us.oracle.com` |
+| `rcuCredentialsSecret` | The Kubernetes secret containing the database credentials. | `soainfra-rcu-credentials` |
+
+In addition, you should make a note of the environment variable `DOMAIN_TYPE` in the `create-domain-job-template.yaml` file, in the same directory as the `create-domain-inputs.yaml` file.
+For SOA domains, by default, the value is `soa` (lower case). You must update this to `osb` for OSB and `bpm` for BPM domain types.
+
+Note that the names of the Kubernetes resources in the generated YAML files may be formed with the
+value of some of the properties specified in the `create-inputs.yaml` file. Those properties include
+the `adminServerName`, `clusterName` and `managedServerNameBase`. If those values contain any
+characters that are invalid in a Kubernetes service name, those characters are converted to
+valid values in the generated YAML files. For example, an uppercase letter is converted to a
+lowercase letter and an underscore `("_")` is converted to a hyphen `("-")`.
+
+The sample demonstrates how to create a SOA Suite domain home and associated Kubernetes resources for a domain
+that has one cluster only. In addition, the sample provides the capability for users to supply their own scripts
+to create the domain home for other use cases. The generated domain YAML file could also be modified to cover more use cases.
+
+In addition, you should update the generated `domain.yaml` file by performing the following steps.
+
+##### Mandatory Configurations
+
```

Comment:
Please remove lines 237 to 243. These are obsolete now.

Reply:
Wouldn't it make more sense to just remove this section if there are no mandatory configuration changes?